conformance/conformance_test.cc

```diff
@@ -1771,6 +1771,10 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
       R"({"repeatedDuration": ["1.5s", "-1.5s"]})",
       "repeated_duration: {seconds: 1 nanos: 500000000}"
       "repeated_duration: {seconds: -1 nanos: -500000000}");
+  RunValidJsonTest(
+      "DurationNull", REQUIRED,
+      R"({"optionalDuration": "null"})",
```

Comment:
This is not correct. null is a JSON keyword whereas "null" is just a string value with 4 characters. Parsers should accept:
```
{"optionalDuration": null}
```
but the following should be rejected as invalid input:
```
{"optionalDuration": "null"}
```